Xopt-GEECS/environment.yml

```diff
@@ -1,11 +1,12 @@
-name: Xopt-GEECS
+name: Xopt-GEECS-test
```

Comment:
Is this still supposed to be Xopt-GEECS-test ?

Reply:
nope

Reply:
Alright, as a reminder any commits you tack on to this `xopt-geecs-environments-dev` branch, such as correcting this, will be automatically added to the pull request.